docs/robotics/perception/vision.md

```diff
@@ -30,10 +25,30 @@ Furthermore, segments which are above the horizon or overlap the robots limbs ar
 
 Each segment contains it's location, color, edge types, and a pre-calculated classification of field color intensity.
 
+### Image Segments Debug View
+
+![Image Segments Debug View](./segmented_image.jpg)
+
+-   red border: rising edge
+-   blue border: falling edge
+-   golden border: image border
+-   black border: limb border
```

Comment:
These are almost completely invisible in the image, especially when compared to the line detection line renderings.
Maybe find a way to make them bigger or use an image that has a higher density?

If I didn't know they were there, I probably wouldn't have noticed them...

Reply:
Yes, true. I added tabs with zoomed in images